src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json

```diff
@@ -90,59 +90,53 @@
           "DisplayName": "Liquid Page",
           "Hidden": false,
           "Settings": {
```

Comment:
Does anyone know what this `Hidden` setting is? 
I can't find any reference to it in the code.
Wonder if it's old and should be removed?